src/postgraphile/withPostGraphileContext.ts

```diff
@@ -277,8 +277,8 @@ function getSettingsForPgClientTransaction({
   pgSettings,
 }: {
   jwtToken?: string;
-  jwtSecret?: string | Buffer;
-  jwtPublicKey?: string | Buffer;
+  jwtSecret?: jwt.Secret;
```

Comment:
Should we be doing the same for `jwtSecret`?

Reply:
If you mean `| GetPublicKeyOrSecret`, that doesn't work as [`jwt.sign`](https://github.com/auth0/node-jsonwebtoken#jwtsignpayload-secretorprivatekey-options-callback) does not accept a callback but requires the secret to be available synchronously. That's why I was really happy to see that @speller had just separated the types…

Reply:
:+1: Cool; always good to have justification. :)